cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/LowestAddressJoinDecider.scala

```diff
@@ -74,7 +74,8 @@ class LowestAddressJoinDecider(system: ActorSystem, settings: ClusterBootstrapSe
                 lowestAddress.map(contactPointString).getOrElse(""),
                 info.contactPoints.map(contactPointString).mkString(", ")
               )
-            else
+          } else {
+            if (log.isWarningEnabled)
```

Comment:
Strictly speaking, checking if warning logging is enabled is not that useful. Warn logging is often enable so the chances of optimisation are minimal here. That's much different that `info` and `debug`. Those are the levels that are in practice tweaked by users.

Reply:
Makes sense.

Reply:
Anyway, not a big issue and I think it's ok to leave like that. That was just a side commend.